watertap/property_models/anaerobic_digestion/adm1_properties_P.py

```diff
@@ -0,0 +1,453 @@
+#################################################################################
```

Comment:
I figured we would discuss how to implement this first, but I guess this is just a draft. Have you thought about inheriting from the existing adm1 class? If there is a lot of the same code, maybe we don't need to copy paste everything.

Reply:
The only part of the code that should be the same in the end are the list of stoichiometric parameters (although there are a few additions to this list). So perhaps it is worth inheriting these from ADM1. But to play devil's advocate, it'd make the code easier to follow if all the parameters were listed out in this file as well.